site/src/main/java/uk/nhs/digital/indices/IsStickySectionMainHeading.java

```diff
@@ -0,0 +1,25 @@
+package uk.nhs.digital.indices;
```

Comment:
Well done. Good to know we can implement Freemarker functions in Java.

Reply:
This was a real breakthrough in getting this ticket done. I will defiantly be a handy tool in the future too.